webgpu.h

```diff
@@ -806,11 +807,16 @@ typedef struct WGPUBlendState {
     WGPUBlendComponent alpha;
 } WGPUBlendState;
 
+typedef struct WGPUCompilationInfo {
+    uint32_t messageCount;
+    WGPUCompilationMessage const * messages;
```

Comment:
WGPUCompilationMessage isn't defined yet, and this isn't used

Reply:
Whoops